.markdownlint.yaml

```diff
@@ -142,3 +142,6 @@ link-image-style:
 # Use pipes on both ends of every line in a table.
 table-pipe-style:
   style: leading_and_trailing
+
+no-duplicate-heading:
+  siblings_only: true
```

Comment:
All other settings have a descriptive comment.

Reply:
Oh right, I pushed this quick. Will update it